add-on/panels/js/sidebar.jsx

```diff
@@ -148,22 +165,20 @@ loop.sidebar = (function(mozL10n) {
       });
 
       ReactDOM.render(<SidebarControllerView
-                        activeRoomStore={activeRoomStore}
                         dispatcher={dispatcher} />, document.querySelector("#main"));
 
-      var locationData = sharedUtils.locationData();
-      var hash = locationData.hash.match(/#(.*)/);
-
       dispatcher.dispatch(new sharedActions.SetupWindowData({
-        windowId: "id-test",
-        roomToken: hash[1],
-        type: "room"
+        roomToken: roomToken
       }));
+
+      loop.request("TelemetryAddValue", "LOOP_ACTIVITY_COUNTER", constants.LOOP_MAU_TYPE.OPEN_CONVERSATION);
     });
   }
 
   return {
-    init: init
+    DesktopSidebarView: DesktopSidebarView,
```

Comment:
Please can you either add an XXXakita  and empty `describe("DesktopSidebarView", function() { it.skip("should have tests")})` in the tests, or write tests for this view please.